dymos/transcriptions/explicit_shooting/test/test_explicit_shooting.py

```diff
@@ -250,52 +195,67 @@ def test_brachistochrone_explicit_shooting_path_constraint(self):
                 phase.set_state_options('y', fix_initial=True)
                 phase.set_state_options('v', fix_initial=True)
 
-                phase.add_parameter('g', val=9, units='m/s**2', opt=True, lower=9, upper=9.80665)
-                phase.add_control('theta', val=45.0, units='deg', opt=True, lower=1.0E-6, upper=179.9)
+                phase.add_parameter('g', val=1.0, units='m/s**2', opt=True, lower=1, upper=9.80665)
```

Comment:
Where are we, the moon or something?

Reply:
Somewhere where we can choose the gravity acting on us.  So a rotating space station or something.  Andrew Wiggin's brachistochrone.